requirements.txt

```diff
@@ -1,5 +1,4 @@
-jaxlib==0.4.26+cuda12.cudnn89
-jax==0.4.26
+jax[cpu]
```

Comment:
can you swap this to `jax[cuda12]` and put this in the `requirements-cpu.txt` file?